src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Random random = new Random();
+        int randomNumber = random.nextInt(100);
```

Comment:
100 should be a class level constant, use constant case for naming

Reply:
thank you! I corrected all the mistakes and performed a new push for you to review